Explorer/Assets/DCL/Navmap/EventInfoPanelController.cs

```diff
@@ -0,0 +1,198 @@
+
+        private void OpenUrl(string url) =>
+            webBrowser.OpenUrl(url);
+
+        private void AddRecurrentEvents(EventDTO @event)
+        {
+            DateTime.TryParse(@event.next_start_at, null, DateTimeStyles.RoundtripKind, out DateTime nextStartAt);
+
+            foreach (string dateStr in @event.recurrent_dates)
+            {
+                if (!DateTime.TryParse(dateStr, null, DateTimeStyles.RoundtripKind, out DateTime date)) continue;
+                if (date < nextStartAt) continue;
+
+                EventScheduleElementView element = scheduleElementPool.Get();
+
+                // TODO: we might need to convert to local, currently R:RFC1123 Fri, 18 Apr 2008 20:30:00 GMT
+                element.DateLabel.text = date.ToString("R");
+
+                element.AddToCalendarButton.onClick.AddListener(() => AddRecurrentEventToCalendar(date));
+                scheduleElements.Add(element);
+            }
+        }
+
+        private void AddRecurrentEventToCalendar(DateTime startAt)
+        {
+            // Same link as https://decentraland.org/events/event?id=... website
+            var description = $"jump in: https://play.decentraland.org/?position={@event?.x},{@event?.y}";
+
+            DateTime nextStartAt = DateTime.Parse(@event?.next_start_at, null, DateTimeStyles.RoundtripKind);
+            DateTime nextFinishAt = DateTime.Parse(@event?.next_finish_at, null, DateTimeStyles.RoundtripKind);
+            TimeSpan duration = nextFinishAt - nextStartAt;
+
+            userCalendar.Add(@event?.name, description, startAt, startAt + duration);
+        }
+
+        private void ClearScheduleElements()
+        {
+            foreach (EventScheduleElementView scheduleElement in scheduleElements)
+            {
+                scheduleElement.AddToCalendarButton.onClick.RemoveAllListeners();
+                scheduleElementPool.Release(scheduleElement);
+            }
+
+            scheduleElements.Clear();
+        }
+
+        private void SetInterested(bool interested)
+        {
+            interestedCancellationToken = interestedCancellationToken.SafeRestart();
+            SetInterestedAsync(interestedCancellationToken.Token).Forget();
+            return;
+
+            async UniTaskVoid SetInterestedAsync(CancellationToken ct)
+            {
+                if (interested)
+                    await eventsApiService.MarkAsInterestedAsync(@event?.id, ct);
+                else
+                    await eventsApiService.MarkAsNotInterestedAsync(@event?.id, ct);
+
+                interestedButtonController.SetButtonState(interested);
+            }
+        }
+
+        private void Share()
+        {
+            shareContextMenu.Set(@event!.Value);
+            shareContextMenu.Show(view.SharePivot);
+        }
+
+        private void JumpIn()
+        {
+            navmapBus.JumpIn(place!);
+            chatMessagesBus.Send($"/{ChatCommandsUtils.COMMAND_GOTO} {@event?.x},{@event?.y}", "jump in");
```

Comment:
This doesnt look like the correct `ORIGIN`. Shouldnt it be something like `event`? Maybe check with the data team what they expect.

Reply:
It is correct as the `@event` is the info of the event which has the coordinates where it is happening